### PR TITLE
feat(content):  add locales en, fr

### LINKED ISSUE #138: 🚀 [Feature] Add Locales EN, FR
## 🚀 Feature

Add Languages EN, FR.

**English translation:** 
[ocelot.social-website-content-en.odt](https://github.com/Ocelot-Social-Community/ocelot.social/files/15418377/ocelot.social-website-content-en.odt)

**French translation:**
[ocelot.social-website-content-fr.odt](https://github.com/Ocelot-Social-Community/ocelot.social/files/15418724/ocelot.social-website-content-fr.odt)

### Todo

- [x] German: Footer: Chance "Authoren" to "Autoren" or "Freunde"
- [x] donate – care for @Elweyn suggestions:
  - [x] https://github.com/Ocelot-Social-Community/ocelot.social/pull/55#discussion_r1530209921
  - [x] https://github.com/Ocelot-Social-Community/ocelot.social/pull/55#discussion_r1530221074
- [ ] description, see https://github.com/Ocelot-Social-Community/ocelot.social/issues/26#issuecomment-2126660467

#### DevOps

- [ ] there is a short blink of `404` on URL `http://localhost:8080/` before it switches to `http://localhost:8080/de/`
- [ ] do we have to set again `redirect: { autoLocale: true }` ?
- [ ] #139

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Add Languages EN, FR.

**English translation:** 
[ocelot.social-website-content-en.odt](https://github.com/Ocelot-Social-Community/ocelot.social/files/15418377/ocelot.social-website-content-en.odt)

**French translation:**
[ocelot.social-website-content-fr.odt](https://github.com/Ocelot-Social-Community/ocelot.social/files/15418724/ocelot.social-website-content-fr.odt)

### Issues

- fixes #138 

### Todo

- [x] remaining translations:
  - [x] search for `XXX translate`
  - [x] `wiki` small caps or capitals in EN and FR?
- [ ] functions: translate images on home #146 
- [x] German: Footer: Chance "Authoren" to "Autoren" or "Freunde"
- [x] donate – care for @Elweyn suggestions:
  - [x] https://github.com/Ocelot-Social-Community/ocelot.social/pull/55#discussion_r1530209921
  - [x] https://github.com/Ocelot-Social-Community/ocelot.social/pull/55#discussion_r1530221074
- [x] description, see https://github.com/Ocelot-Social-Community/ocelot.social/issues/26#issuecomment-2126660467

#### DevOps

- [ ] there is a short blinking of `404` page on URL `http://localhost:8080/` before it redirects to `http://localhost:8080/de/`, see #143 
- [x] do we have to set again `redirect: { autoLocale: true }` ?
- [ ] #139

### CODE REVIEW

File: docs/fr/README.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    <figcapture>
+      <h3>Évolutif</h3>
+      Grâce à la technologie la plus récente, tu peux utiliser le logiciel pour toutes les tailles de réseau.
+    </figcapture>
+  </figure>
+  <figure>
+    <HopeIcon icon="home" />
+    <figcapture>
+      <h3>Hébergement flexible</h3>
+      Tu peux héberger votre réseau vous-même ou le faire héberger par votre fournisseur de services.
```
Comment (Elweyn) [inline docs/fr/README.md:82]: Duzen und siezen im gleichen satz => Duzen
```suggestion
      Tu peux héberger ton réseau toi-même ou le faire héberger par ton fournisseur de services.
```

File: docs/fr/README.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+  </figure>
+</div>
+
+<h2 id="feature-headline" class="large-header">Fonctions les plus importantes</h2>
+
+<div class="image-grid center">
+  <figure>
+    <img src="../de/assets/login.png" alt="Aperçu des messages (newsfeed)" />
+    <figcapture>
+      Aperçu des messages (newsfeed)
```
Comment (Elweyn) [inline docs/fr/README.md:107]:
```suggestion
      Connexion
```

File: docs/fr/README.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/newsfeed.png" alt="Post overview (newsfeed)" />
+    <figcapture>
+      Post overview (newsfeed)
```
Comment (Elweyn) [inline docs/fr/README.md:113]:
```suggestion
      Aperçu des contributions (newsfeed)
```

File: docs/de/README.md [hunk 4]
```diff
@@ -195,6 +195,39 @@ tagLine:
...
+  </figure>
+  <figure>
+    <HopeIcon icon="fa-solid" />
+    <figcapture>
+      <h3>Veränderbar</h3>
+      Jede Person kann etwas zu ihrem Lieblingsprogramm beitragen, jederzeit. Ich kann das Programm so verwenden, wie ich es am liebsten will.
+    </figcapture>
+  </figure>
+</div>
+
```
Comment (mahula) [inline docs/de/README.md:230]: This part is missing in the English and French versions.

File: docs/fr/README.md [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/single-post.png" alt="Message individuel" />
+    <figcapture>
+      Message individuel
```
Comment (Elweyn) [inline docs/fr/README.md:119]:
```suggestion
      Contribution individuelle
```

File: docs/fr/README.md [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/newsfeed.png" alt="Post overview (newsfeed)" />
+    <figcapture>
+      Post overview (newsfeed)
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/single-post.png" alt="Message individuel" />
```
Comment (Elweyn) [inline docs/fr/README.md:117]:
```suggestion
    <img src="../de/assets/single-post.png" alt="Contribution individuelle" />
```

File: docs/fr/README.md [hunk 7]
```diff
@@ -0,0 +1,291 @@
...
+
+<div class="image-grid center">
+  <figure>
+    <img src="../de/assets/login.png" alt="Aperçu des messages (newsfeed)" />
+    <figcapture>
+      Aperçu des messages (newsfeed)
+    </figcapture>
+  </figure>
+  <figure>
+    <img src="../de/assets/newsfeed.png" alt="Post overview (newsfeed)" />
```
Comment (Elweyn) [inline docs/fr/README.md:111]:
```suggestion
    <img src="../de/assets/newsfeed.png" alt="Aperçu des contributions (newsfeed)" />
```

File: docs/fr/README.md [hunk 8]
```diff
@@ -0,0 +1,291 @@
...
+      Le logiciel est à la pointe de la technologie – flexible, fiable, performant.
+    </figcapture>
+  </figure>
+</div>
+
+<h2 id="feature-headline" class="large-header">Fonctions les plus importantes</h2>
+
+<div class="image-grid center">
+  <figure>
+    <img src="../de/assets/login.png" alt="Aperçu des messages (newsfeed)" />
```
Comment (Elweyn) [inline docs/fr/README.md:105]:
```suggestion
    <img src="../de/assets/login.png" alt="Connexion" />
```

File: docs/fr/README.md [hunk 9]
```diff
@@ -0,0 +1,291 @@
...
+-->
+
+<div class="center">
+  <p>
+    Testes la démo en direct sur
+    <a href="https://stage.ocelot.social" target="_blank">https://stage.ocelot.social</a>
+    avec les données d’accès suivantes :
+  </p>
+
+  <table><thead><tr><th style="text-align:left;">Courriel</th><th style="text-align:left;">Mot de passe</th><th style="text-align:left;">Rôle de l’utilisateur</th></tr></thead><tbody><tr><td style="text-align:left;"><code>user@example.org</code></td><td style="text-align:left;">1234</td><td style="text-align:left;">utilisateur normal</td></tr><tr><td style="text-align:left;"><code>moderator@example.org</code></td><td style="text-align:left;">1234</td><td style="text-align:left;">modérateur</td></tr><tr><td style="text-align:left;"><code>admin@example.org</code></td><td style="text-align:left;">1234</td><td style="text-align:left;">administrateur</td></tr></tbody></table>
```
Comment (Elweyn) [inline docs/fr/README.md:250]:
```suggestion
  <table>
    <thead>
      <tr>
        <th style="text-align:left;">Courriel</th>
        <th style="text-align:left;">Mot de passe</th>
        <th style="text-align:left;">Rôle de l’utilisateur</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td style="text-align:left;"><code>user@example.org</code></td>
        <td style="text-align:left;">1234</td>
        <td style="text-align:left;">utilisateur normal</td>
      </tr>
      <tr>
        <td style="text-align:left;"><code>moderator@example.org</code></td>
        <td style="text-align:left;">1234</td>
        <td style="text-align:left;">modérateur</td>
      </tr>
      <tr>
        <td style="text-align:left;"><code>admin@example.org</code></td>
        <td style="text-align:left;">1234</td>
        <td style="text-align:left;">administrateur</td>
      </tr>
    </tbody>
  </table>
```

File: docs/fr/README.md [hunk 10]
```diff
@@ -0,0 +1,291 @@
...
+</div>
+
+<h2 class="large-header">Commences</h2>
```
Comment (Elweyn) [inline docs/fr/README.md:253]:
```suggestion
<h2 class="large-header">Commence</h2>
```

File: docs/fr/contribute/README.md [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+### Gére ton propre réseau
+
+Avec le logiciel gratuit *ocelot.social* et le savoir-faire nécessaire, tu peux gérer ton propre réseau (→ [Documentation](https://docs.ocelot.social/deployment/), en anglais).
+
+Si tu asbesoin d’aide, nos développeurs se feront un plaisir de [t’aider](/fr/contact/).
+
+### Faire connaître le logiciel *ocelot.social*
+
+<!-- markdownlint-disable-next-line no-emphasis-as-heading -->
+*«Répandes la bonne parole !»*
```
Comment (Elweyn) [inline docs/fr/contribute/README.md:67]:
```suggestion
*«Répand la bonne parole !»*
```

File: docs/fr/contribute/README.md [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+
+Qu’il s’agisse d’une association, d’un mouvement ou d’une institution, *ocelot.social* peut être utilisé comme un réseau social pour de nombreux types de groupes et d’organisations.
+
+- publies le lien vers [*ocelot.social*](https://ocelot.social) sur ton site web, ton blog ou tes réseaux sociaux
+- donnes un `Like` au logiciel *ocelot.social* sur la plateforme de comparaison [alternative.to](https://alternativeto.net/software/ocelot-social/about/)
```
Comment (Elweyn) [inline docs/fr/contribute/README.md:72]:
```suggestion
- donnes un `J’aime` au logiciel *ocelot.social* sur la plateforme de comparaison [alternative.to](https://alternativeto.net/software/ocelot-social/about/)
```

File: docs/fr/contribute/README.md [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+
+### Aider à développer le logiciel
+
+Le logiciel *ocelot.social* est gratuit et open source.
+Tu souhaites développer et tester de nouvelles fonctions ?
+Suives notre [documentation](https://docs.ocelot.social/CONTRIBUTING.html) en anglais.
+
+Nos [Good First Issues](https://github.com/Ocelot-Social-Community/Ocelot-Social/labels/good%20first%20issue) sur GitHub conviennent aux débutants.
+
+Nous sommes également heureux que tu consulties, commenties ou apporties des modifications au [code de programme](https://github.com/Ocelot-Social-Community/Ocelot-Social/pulls) d’autres développeurs sur GitHub.
```
Comment (Elweyn) [inline docs/fr/contribute/README.md:115]:
```suggestion
Nous sommes également heureux que tu consulte, commente ou apporte des modifications au [code](https://github.com/Ocelot-Social-Community/Ocelot-Social/pulls) d’autres développeurs sur GitHub.
```

File: docs/fr/contribute/README.md [hunk 14]
```diff
@@ -0,0 +1,135 @@
...
+Ou donnez à [software on GitHub](https://github.com/Ocelot-Social-Community/Ocelot-Social) une `Étoile`.
```
Comment (Elweyn) [inline docs/fr/contribute/README.md:116]:
```suggestion
Ou donne une `Étoile` au [code sur GitHub](https://github.com/Ocelot-Social-Community/Ocelot-Social).
```

**Review by mahula — CHANGES_REQUESTED** [review]

File: docs/fr/features/README.md [hunk 15]
```diff
@@ -0,0 +1,116 @@
...
+Les fonctionnalités suivantes ont été intégrées jusqu’à présent :
+
+## Utilisateurs
+
+- enregistrement d’un utilisateur
+- profil de l’utilisateur
+- créer un message
+- suivre l’utilisateur
+- bloquer l’utilisateur
+- couper le son de l’utilisateur
```
Comment (Elweyn) [inline docs/fr/features/README.md:19]: Sounds strange but don't find a better translation

File: docs/fr/imprint/README.md [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+home: false
+# icon: home
+sidebar: false
+title: Impressum
+---
+
+## Responsable de l’exploitation de ce site Web
+
+busFaktor() e.V.  
+Wolfgang Huß, 2ème Président
```
Comment (Elweyn) [inline docs/fr/imprint/README.md:11]:
```suggestion
Wolfgang Huß, vice-président  
```

File: docs/fr/contact/README.md [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+home: false
+# icon: home
+title: Contact
+sidebar: false
+---
+
+## Association des développeurs
+
+busFaktor() e.V.  
+Wolfgang Huß, 2e président
```
Comment (Elweyn) [inline docs/fr/contact/README.md:11]:
```suggestion
Wolfgang Huß, vice-président  
```

File: docs/fr/privacy-policy/README.md [hunk 18]
```diff
@@ -0,0 +1,86 @@
...
+Lors de l’utilisation de notre site web, différents types de données personnelles sont traités de différentes manières. Nous t’indiquons ci-après quelles applications et quels services sont traités, quelles sont les données à caractère personnel concernées, quelle est la finalité du traitement, combien de temps les données sont conservées dans chaque cas et bien d’autres choses encore.
+
+### Site web
+
+Lors de l’utilisation de notre site web, différents types de données à caractère personnel sont traités de différentes manières. Dans ce qui suit, nous t’informons sur les types de traitement, les données personnelles concernées dans chaque cas, la finalité du traitement respectif, la durée de conservation des données dans chaque cas et bien d’autres choses encore.
+
+#### Accès au site web
+
+**Purpose:** We make our website available to the public so that everyone can inform themselves about our association, its activities and offers.
+
```
Comment (Elweyn) [inline docs/fr/privacy-policy/README.md:41]: Duplicate with Objectif
```suggestion
```

**Review by Elweyn — CHANGES_REQUESTED** [review]

File: docs/de/README.md (hunk 4 above)
Comment (Tirokk) [reply docs/de/README.md:230]: That is in development and came in by accident.
I comment it out.

Comment (Tirokk) [reply docs/de/README.md]: Good catch 🙏🏼

**Review by Elweyn — APPROVED** [review]

**Review by mahula — APPROVED** [review]

**Pull request merged**